openfisca_us/entities.py

```diff
@@ -8,6 +8,16 @@
 
 from openfisca_core.entities import build_entity
 
+
+BenefitUnit = build_entity(
+    key="benefitunit",
+    plural="benefitunits",
+    label="The benefit unit",
+    doc="""
+    Description of a benefit unit
+    """,
+)
+
 TaxUnit = build_entity(
```

Comment:
I set taxunits as person-level entities, but eventually we'll probably need to return to them and make them groups with dependents in them?